pkg/types/gcp/platform.go

```diff
@@ -9,6 +9,9 @@ type Platform struct {
 	// Region specifies the GCP region where the cluster will be created.
 	Region string `json:"region"`
 
+	// PublicZoneID specifies the ID for the public DNS Zone
+	PublicZoneID string `json:"publicZoneID"`
```

Comment:
can you define how the ID needs to be structured, or at least an example of such..?

Reply:
I have updated with a comment. Let me know if that is sufficient.

Reply:
the 64-bit unsigned number is not sufficient, it still needs the project-id to fetch the dns zone.

a) https://github.com/openshift/installer/pull/2764/commits/022bb3b62ac673b04568f0911da9adce28ef3289#r382345970
b) let's make sure it's clear that the zone must be in the same zone as cluster.

Reply:
Updated.